locale/translations/el_EL.json

```diff
@@ -152,7 +152,8 @@
     "page.keyboard_shortcuts.open_original_same_window": "Άνοιγμα αρχικού συνδέσμου στην τρέχουσα καρτέλα",
     "page.keyboard_shortcuts.open_comments": "Άνοιγμα συνδέσμου σχολίων",
     "page.keyboard_shortcuts.open_comments_same_window": "Άνοιγμα συνδέσμου σχολίων στην τρέχουσα καρτέλα",
-    "page.keyboard_shortcuts.toggle_read_status": "Εναλλαγή ανάγνωσης / μη αναγνωσμένης",
+    "page.keyboard_shortcuts.toggle_read_status_next": "Εναλλαγή ανάγνωσης / μη αναγνωσμένης, εστίαση στη συνέχεια",
```

Comment:
By the way, outside of scope of this PR, this locale is actually greek. I do not know where `el_EL` comes from, I expected it should have been `el_GR`.